### PR TITLE
Fran0571 _buttons feature completed.

### PR DESCRIPTION
**Pull request by fran0571**
Please @le000255 check the final changes to the original code that include:
1. Replacing JavaScript code with HTML tags.
2. Removing the hover state from disabled buttons.

### CODE REVIEW

File: src/_buttons.scss [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  "secondary": $fe-clr-secondary,  //accordign to the draft it should be purple but that color code is not in the globalvariables.
+  "success": $fe-clr-success,
+  "info": $fe-clr-info,
+  "warning": $fe-clr-warning,
+  "danger": $fe-clr-danger,
+  "default": $fe-clr-dark,
+);
+
+//Create a mixin for buttons
+@mixin btn() {
```
Comment (le000255) [inline src/_buttons.scss]: I suggest moving this mixin to `_mixins.scss`

**Review by le000255 — CHANGES_REQUESTED** [review]
Hi Johely, great work!

I just have a minor suggestion that you move the mixin to `_mixins.scss` file. 

After doing it, you can push the change to your own branch. I'll see what happen. 
LOL

File: src/_buttons.scss (hunk 1 above)
Comment (fran0571) [reply src/_buttons.scss]: OK, thank you. 
I moved the mixins code from the _buttons to _mixins file. 
Already pushed the changes.

**Review by le000255 — APPROVED** [review]

**Comment by le000255** [comment]
Approved!

**Pull request merged**